RELEASING.md

```diff
@@ -8,7 +8,7 @@ Releasing checklist:
 
 ```cairo
 # SPDX-License-Identifier: MIT
-# OpenZeppelin Contracts for Cairo v0.1.0 (account/Account.cairo)
+# OpenZeppelin Contracts for Cairo v0.2.0 (account/Account.cairo)
```

Comment:
👍

Reply:
haha NO! this should not have been there

```suggestion
# OpenZeppelin Contracts for Cairo v0.1.0 (account/Account.cairo)
```